os/arch/arm/src/armv7-m/mpu.h

```diff
@@ -146,6 +148,7 @@
 #define MPU_RASR_S              (1 << 18)	/* Bit 18: Shareable */
 #define MPU_RASR_TEX_SHIFT    (19)	/* Bits 19-21: TEX Address Permisson */
 #define MPU_RASR_TEX_MASK     (7 << MPU_RASR_TEX_SHIFT)
+#define MPU_RASR_TEX(n)		((uint32_t)(n) << MPU_RASR_TEX_SHIFT)
```

Comment:
Please aline macros.

Reply:
there is a single tab for the change made